extensions/amp-sidebar/1.0/component.js

```diff
@@ -215,20 +220,24 @@ export function SidebarToolbar({
 
     const selector = `#${CSS.escape(toolbarTarget)}`;
     const newTargetEl = document.querySelector(selector);
+    console.log('targetEl', newTargetEl, selector);
     setTargetEl(newTargetEl);
   }, [toolbarTarget]);
 
+  console.log('matches', matches, targetEl, children, rest);
+
   return (
     <>
-      <nav
+      <Comp
         ref={ref}
         toolbar={mediaQueryProp}
         toolbar-target={toolbarTarget}
         {...rest}
       >
         {children}
-      </nav>
+      </Comp>
       {matches && targetEl && createPortal(children, targetEl)}
+      {matches && targetEl && createPortal('children', targetEl)}
```

Comment:
Right now only the second one of these `"children"` is rendered to the target element.  I am wondering if I am passing the `children` prop incorrectly.  It's currently passed from `child.children` where `child` is the `<nav />` element from the amp-side.

Reply:
TBH I'm really confused here. There's still a duplicate `children` output one line above, and I don't understand here how the `'children'` (as a string) would work here either.